analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/NullPointerDereferenceTest.cs

```diff
@@ -25,11 +25,13 @@ namespace SonarAnalyzer.UnitTest.Rules.SymbolicExecution
     [TestClass]
     public class NullPointerDereferenceTest
     {
-        private static readonly DiagnosticDescriptor[] OnlyDiagnostics = new[] { NullPointerDereference.S2259 };
+        private readonly VerifierBuilder verifierSonar = new VerifierBuilder<SymbolicExecutionRunner>()
```

Comment:
For simplicity, so the first word says if it's `sonar` or `roslyn`
```suggestion
        private readonly VerifierBuilder sonar = new VerifierBuilder<SymbolicExecutionRunner>()
```

It could also be `sonarSE`, but it's quite clear to me what the difference between `sonar` and `roslyn` would be.